packages/dts-generator/lib/phases/ast-fixer.js

```diff
@@ -273,6 +273,7 @@ function fixNamespacesAsInterfaces(
         name: nestedNs.name,
         methods: nestedNs.functions,
         parent: currNS,
+        namespace: true,
```

Comment:
Maybe a more descriptive name? e.g;
- `originallyNamespace` or something similar?

Reply:
I think the d.ts of the AST should also be updated.
- https://github.com/SAP/ui5-typescript/blob/master/packages/dts-generator/types/ast.d.ts#L83-L90